tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenRunCommandTest.cs

```diff
@@ -65,23 +67,29 @@ protected override void CallMethod(CancellationToken cancellationToken)
         {
             if (_session == null)
             {
-                _result = _database.RunCommand<BsonDocument>(_command, _readPreference, cancellationToken);
+                _result = _database.WithReadConcern(_readConcern).RunCommand<BsonDocument>(_command, _readPreference, cancellationToken);
```

Comment:
Make multiline to be consistent with your other changes?

Reply:
Fixed!